docs/dev/new_check_howto.md

```diff
@@ -412,3 +414,4 @@ python setup.py bdist_wheel
 [12]: https://docs.datadoghq.com/integrations/
 [13]: https://www.uuidgenerator.net/
 [14]: https://docs.datadoghq.com/getting_started/tagging/
+[15]: https://github.com/DataDog/integrations-core/tree/e4f5ba647ba0d6d81e0598763dffefe5a4464af2/datadog_checks_dev#development
```

Comment:
Why does this link to a specific commit and not [HEAD](https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev#development)?

Reply:
@cswatt here https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev#development